packages/contracts/src/2.0.0/protocol/Exchange/libs/LibMath.sol

```diff
@@ -29,7 +29,7 @@ contract LibMath is
     /// @param numerator Numerator.
     /// @param denominator Denominator.
     /// @param target Value to calculate partial of.
-    /// @return Partial value of target.
+    /// @return Partial value of target rounded down.
```

Comment:
If we change this to `getPartialAmountFloor` then it forces developers to consider the direction of rounding.